### DISCUSSION TITLE
辅助管理权限说明

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#477] [公告]
辅助管理权限可以允许你修改 issue 的标签，在一定程度上管理 issue 和 discussion。
现已不再接受主动的权限申请，我会时不时观察一下 issue 和 discussion， 如果发现合适的用户的话，我会主动给他添加权限的。
这当然也不是强制的，没有 “任务要求”，有空了就看看，没空不管也完全无所谓，放轻松……如果不想要权限，也可以直接忽略。

### 注意
是否给予辅助管理权限取决于是否想并且有能力帮助 PCL 社区。它是让你能更好地帮助社区的 **工具**，绝非你帮助社区的 **目的** 。我只会把它给予需要它来帮助社区的人，而非想用它来彰显自己的人。它不是一个 “成就” 或者 “奖章”，这里也不是贴吧或者积分制论坛。如果你的行为是以获取权限为根本目的，我绝对不会给你权限。
请不要以此为目的去提交 issue 或者去评论，不要认为所谓的 “活跃” 就能让你拿到权限。有些人大量提交 issue，但大部分最后都被拒绝，或者它压根没到需要开 issue 或者 PR 的程度。此类行为只会为大家和我带来额外的工作量，对社区的坏处远大于好处。不要这样，谢谢大家。

---

如果你拥有权限，下面是你能帮上忙的事！

## Issues

所有的新反馈都自动带有 `新提交` 标签。查看这些反馈，**先移除** `新提交` **标签** ，再根据情况：
- **和已有反馈重复：** 回复重复反馈的编号，并添加 `重复` 标签
- **与 PCL 无关：** 例如 MC 进不去服务器，不会安装 Mod 等。回复建议其在 [提问板块](https://github.com/Hex-Dragon/PCL2/discussions/categories/%E6%8F%90%E9%97%AE) 发帖，并添加 `忽略` 标签
- **意义不明的或是纯骂人的：** 添加 `忽略` 标签，如果是纯骂人的或很恶劣的可以添加 `删除` 标签
- **缺少信息：** 例如崩溃未上传错误报告，Bug 未上传日志等，回复具体情况并添加 `等待提交者` 标签
- **正常：** 添加 `等待确认` 标签，代表已通过初步确认，开发者就会继续处理（如果是 Bug 反馈的话最好确认一下可以复现）

点击 [这个链接](https://github.com/Hex-Dragon/PCL2/issues?q=label%3A新提交%2C需要社区复现%2C需要社区帮助%2C社区处理中+) 以查看所有需要社区帮帮忙的 issue。

## Discussions

如果里面有恶意辱骂等不正常帖子，可以先警告，然后锁定或删除。

看一看 [提问板块](https://github.com/Hex-Dragon/PCL2/discussions/categories/%E6%8F%90%E9%97%AE) 有没有能回复的。

---

多谢大家的帮助和支持！

**Comment by Louis-Harsune** [comment]
这个“比较活跃”有没有什么具体标准啊…….jpg

**Reply by quannen** [reply]
我也很好奇.png

**Reply by LTCatt** [reply]
我会看 Issue 和 Discussion 的回复情况，还有个人 GitHub 账号的活跃情况，来综合考虑 =。=

**Reply by LTCatt** [reply]
简单地说就是我会看 Issue 和 Discussion 的回复量和质量，以及 GitHub 账号本身是否活跃，如果都可以我就会直接发权限……

**Comment by iPodtouch6557** [comment]
如何换标签啊）

**Reply by WForst-Breeze** [reply]
只有 Collaborator 和 Member 才有权限更改 label，可以在右侧 Labels 栏中单击齿轮直接进行更改~

**Comment by RawDiamondMC** [comment]
突然发现label改了
那么文章里的[这个链接](https://github.com/Hex-Dragon/PCL2/issues?q=is%3Aopen+is%3Aissue+label%3A%E6%96%B0%E5%8F%8D%E9%A6%88%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%B8%AE%E5%8A%A9%2C%E9%9C%80%E8%A6%81%E8%A1%A5%E5%85%85%E4%BF%A1%E6%81%AF+)是不是也应该改了？
~需要补充信息改成什么了，等待反馈者吗？~

**Reply by WForst-Breeze** [reply]
~其实早就改了（）~ `需要补充信息` 更改为了 `等待反馈者` ，`需要社区帮助` 更改为了 `需要帮助`

**Reply by RawDiamondMC** [reply]
> ~其实早就改了（）~ `需要补充信息` 更改为了 `等待反馈者` ，`需要社区帮助` 更改为了 `需要帮助`

那这个链接应该要改了...

**Comment by RawDiamondMC** [comment]
https://github.com/Hex-Dragon/PCL2/issues?q=is%3Aopen+is%3Aissue+label%3A新反馈%2C需要帮助%2C等待反馈者+
改成这个链接吧

**Reply by LTCatt** [reply]
ok，改了 √

**Comment by so1ve** [comment]
普通用户没有更改label的权限（）

**Comment by MoYuan-CN** [comment]
> 缺少信息： 例如崩溃未上传错误报告，Bug 未上传日志等，回复具体情况并添加 `等待反馈者` 或 `需要补充信息` 标签

`需要补充信息` 过期了（没了）

**Reply by LTCatt** [reply]
已更新 √

**Comment by SqrtTan** [comment]
@LTCatt 普通用户似乎不能改Label的……没那个权限……

**Reply by WorldHim** [reply]
主要是一些 Label 是标记给开发者看的，普通用户要是有权限的话乱改很容易出问题

**Comment by SALTWOOD** [comment]
Label 改了
改一下链接吧

[这个](https://github.com/Hex-Dragon/PCL2/issues?q=is%3Aopen+is%3Aissue+label%3A%E6%96%B0%E5%8F%8D%E9%A6%88%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%A4%8D%E7%8E%B0%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%B8%AE%E5%8A%A9%2C%E7%AD%89%E5%BE%85%E5%8F%8D%E9%A6%88%E8%80%85+)

**Reply by LTCatt** [reply]
改成 [这个](https://github.com/Hex-Dragon/PCL2/issues?q=is%3Aissue+label%3A%E6%96%B0%E5%8F%8D%E9%A6%88%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%A4%8D%E7%8E%B0%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%B8%AE%E5%8A%A9%2C%E7%AD%89%E5%BE%85%E5%8F%8D%E9%A6%88%E8%80%85+) 了

**Reply by MoYuan-CN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/6c8bafe7-0b22-4561-8dfc-30f3b1342347)
emmmm

**Comment by allMagicNB** [comment]
最近是不是没怎么添加协作者啊……（

**Reply by iPodtouch6557** [reply]
啊对对对

**Reply by WForst-Breeze** [reply]
需要的时候龙猫自然会去邀请的，不必着急

**Reply by allMagicNB** [reply]
> 需要的时候龙猫自然会去邀请的，不必着急

看着又是#3092 却无能为力的我：😅
~~看到另一个协作者还贴错了重复的我：？~~

**Reply by WorldHim** [reply]
> > 需要的时候龙猫自然会去邀请的，不必着急
> 
> 看着又是#3092 却无能为力的我：😅
> ~~看到另一个协作者还贴错了重复的我：？~~

你这…太功利了吧🤐

**Reply by allMagicNB** [reply]
> > > 需要的时候龙猫自然会去邀请的，不必着急
> > 
> > 看着又是#3092 却无能为力的我：😅
> > ~~看到另一个协作者还贴错了重复的我：？~~
> 
> 你这…太功利了吧🤐

~~没看懂功利的意思，但是好像是个贬义词？~~

**Reply by hejiehao** [reply]
差不多就是追求名誉的意思

**Reply by allMagicNB** [reply]
> > > 需要的时候龙猫自然会去邀请的，不必着急
> > 
> > 看着又是#3092 却无能为力的我：😅
> > ~~看到另一个协作者还贴错了重复的我：？~~
> 
> 你这…太功利了吧🤐

额，不会吧……主要是想帮帮忙，然后已经确认是重复了，又不能改Label
怎么说呢……

**Reply by SALTWOOD** [reply]
> > > 需要的时候龙猫自然会去邀请的，不必着急
> > 
> > 
> > 看着又是#3092 却无能为力的我：😅
> > ~看到另一个协作者还贴错了重复的我：？~
> 
> 你这…太功利了吧🤐

话不能这么说，人家也可能只是想帮忙罢了。~而且他的话也没体现他功利吧~

**Reply by WorldHim** [reply]
> > > > 需要的时候龙猫自然会去邀请的，不必着急
> > > 
> > > 
> > > 看着又是#3092 却无能为力的我：😅
> > > ~看到另一个协作者还贴错了重复的我：？~
> > 
> > 你这…太功利了吧🤐
> 
> 话不能这么说，人家也可能只是想帮忙罢了。~而且他的话也没体现他功利吧~

我的意思只是说没必要纠结于是不是协作者，能帮忙测试或解答问题就行了
像是确认问题存在与是否重复这种工作每个人都可以做，无非只是协作者可以直接 close 而已
一个社区的发展是每个人共同作用的结果，不仅仅是贡献者、协作者

**Reply by allMagicNB** [reply]
> 我的意思只是说没必要纠结于是不是协作者，能帮忙测试或解答问题就行了

 ~~测试解答完了一个协作者都不在~~

**Reply by SALTWOOD** [reply]
> > > 你这…太功利了吧🤐
> > 
> > 
> > 话不能这么说，人家也可能只是想帮忙罢了。~而且他的话也没体现他功利吧~
> 
> 我的意思只是说没必要纠结于是不是协作者，能帮忙测试或解答问题就行了 像是确认问题存在与是否重复这种工作每个人都可以做，无非只是协作者可以直接 close 而已 一个社区的发展是每个人共同作用的结果，不仅仅是贡献者、协作者

那确实是

**Reply by Deep-Dark-Forest** [reply]
对的，感觉你和[凌云](https://github.com/Jinghai-Lingyun)完全符合这里说的所有要求
~关于我之前把contibutor看成Collaborator然后认为凌云已经是协作者了这件事~

**Reply by ghost** [reply]
和你一样把贡献者认成了协作者

**Comment by ghost** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/110439412/bd62e23d-97f0-46ee-8563-30fefd377104)
邮箱炸了。。。。

**Reply by MoYuan-CN** [reply]
我1.27-2.7没看，1.3k多...

**Reply by allMagicNB** [reply]
> 我1.27-2.7没看，1.3k多...

放寒假了大家不上学就玩Minecraft呗，玩Minecraft又要用到PCL2启动器呗，用PCL2启动器就难免出问题呗，出问题又不排查检查项直接交Issue呗（

**Reply by Pigeon0v0** [reply]
可以设置不进行邮箱提醒

毕竟我有 GitHub Mobile...

**Reply by Deep-Dark-Forest** [reply]
Off-topic，启动！

**Reply by allMagicNB** [reply]
> 可以设置不进行邮箱提醒
> 
> 毕竟我有 GitHub Mobile...

我也有（
但是不知道为什么好像只有@我才会发通知

**Reply by iPodtouch6557** [reply]
我从来不看邮箱消息，都直接开网页看

**Reply by hejiehao** [reply]
我更加喜欢通过邮箱提醒的这种形式

**Reply by allMagicNB** [reply]
> 我更加喜欢通过邮箱提醒的这种形式

我邮箱提醒，GitHub Mobile查看……

**Reply by hejiehao** [reply]
说个题外话
新 Outlook 其实是个浏览器
不过相比其他的 Electron 程序，它没有给你往系统里多塞一个 Chrome 而是直接调用 Webview

**Reply by allMagicNB** [reply]
> 说个题外话
> 新 Outlook 其实是个浏览器
> 不过相比其他的 Electron 程序，它没有给你往系统里多塞一个 Chrome 而是直接调用 Webview

好歹直接调用的都是系统自带的Webview，没有自己集成，节省了空间（

**Reply by ghost** [reply]
然后你会发现，edge带了一份，微软又拷贝了一份，两者是等价的，哪一份都是可以用的，没区别甚至崩溃了还要备份一下

**Comment by allMagicNB** [comment]
不对啊，Collaborator 没法锁帖吧（

**Reply by WForst-Breeze** [reply]
是的

**Reply by allMagicNB** [reply]
> 是的

![Screenshot_20240306_210802.jpg](https://github.com/Hex-Dragon/PCL2/assets/116283787/9566f7a2-9dae-4a5f-99bd-fea79e9c12ac)
:thinking:

**Reply by MoYuan-CN** [reply]
可以Lable删除，但是没法锁帖

**Reply by Deep-Dark-Forest** [reply]
![Screenshot_20240307_205111](https://github.com/Hex-Dragon/PCL2/assets/131975043/09c439fb-31bc-43d9-8b90-692f92251437)
？

**Reply by WForst-Breeze** [reply]
最亮的信标是 Maintainer

**Reply by allMagicNB** [reply]
> 最亮的信标是 Member

不是 Member 吧，好像是维护者（

**Comment by SYBass** [comment]
？居然有这个公告，差点就没看到

**Comment by Deep-Dark-Forest** [comment]
可以考虑给比较稳妥的Collaborator发Maintainer，不然每次锁帖都得找最亮的信标

**Reply by allMagicNB** [reply]
@LTCatt

**Reply by Deep-Dark-Forest** [reply]
想了想感觉不太行
maintainer拥有除删库和访问security以外的所有权限，甚至可以直接向存储库提交文件，有点危险
还是做个添加后就可以让六角龙机器人自动锁帖的标签吧
~或者财大气粗的龙猫去开通个github企业版自定义用户权限~

**Reply by Ad-closeNN** [reply]
啊？Collaborator不能锁帖的吗...为啥测试用的存储库能lock
难道是龙猫改权限了？
~~还是因为这是组织才不能~~
<details>
<summary>
图有点多
</summary>

![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/3e58642e-6635-4c1d-969a-802356068b11)

![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/b5fd97b6-27ca-40ca-b36b-9de7e2b96a00)

![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/69f5d800-1624-47eb-903d-a79c86c7aa94)

![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/764a9d93-4b73-4df5-a47f-6d5181f35c88)

</details>

**Reply by Deep-Dark-Forest** [reply]
个人存储库和组织存储库不一样？可能是
![image](https://github.com/Hex-Dragon/PCL2/assets/131975043/efd81d0e-89ec-4a3f-b0a2-db8b1e182360)
龙猫选的好像是这个

**Reply by Light-Beacon** [reply]
绝对不可能 权限太危险了

**Reply by LTCatt** [reply]
会加个 Lock 权限 :/

**Reply by allMagicNB** [reply]
事实证明，这没必要。

**Comment by 3gf8jv4dv** [comment]
`等待反馈者` → `等待提交者`

---

[讨论时的常见疑惑 · Hex-Dragon/PCL2 · Discussion #775](https://github.com/Hex-Dragon/PCL2/discussions/775)

[暂缓处理](https://github.com/Hex-Dragon/PCL2/issues?q=is%3Aissue+is%3Aopen+label%3A%E6%9A%82%E7%BC%93%E5%A4%84%E7%90%86) looks invalid.

**Reply by LTCatt** [reply]
已修改 :D

**Reply by Deep-Dark-Forest** [reply]
Now “投票中” takes the place of “暂缓处理”.
https://github.com/Hex-Dragon/PCL2/issues/2981

**Comment by Ad-closeNN** [comment]
[这个链接](https://github.com/Hex-Dragon/PCL2/issues?q=label%3A%E6%96%B0%E5%8F%8D%E9%A6%88%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%A4%8D%E7%8E%B0%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%B8%AE%E5%8A%A9%2C%E7%A4%BE%E5%8C%BA%E5%A4%84%E7%90%86%E4%B8%AD+)

应该改为[这个链接](https://github.com/Hex-Dragon/PCL2/issues?q=label%3A%E6%96%B0%E6%8F%90%E4%BA%A4%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%A4%8D%E7%8E%B0%2C%E9%9C%80%E8%A6%81%E7%A4%BE%E5%8C%BA%E5%B8%AE%E5%8A%A9%2C%E7%A4%BE%E5%8C%BA%E5%A4%84%E7%90%86%E4%B8%AD+)?

`新反馈` -> `新提交`

**Reply by LTCatt** [reply]
已修改，多谢
